src/Solr/Implementor/ProductRepository.php

```diff
@@ -23,9 +23,11 @@
     public function getProductsInChunks($storeId, ProductIdChunks $chunks);
 
     /**
+     * @param null|int $sliceId
+     * @param null|int $totalNumberSlices
      * @return int[]
      */
-    public function getAllProductIds();
+    public function getAllProductIds($sliceId = null, $totalNumberSlices = null);
```

Comment:
Are you sure? You can still use this method as before if you skip the parameters. I'd see that as "new feature", but I don't see the BC break.